packages/core/src/components/ButtonGroup/ButtonGroup.module.scss

```diff
@@ -63,3 +63,7 @@
   opacity: 0.5;
   cursor: not-allowed;
 }
+
+.stretch {
```

Comment:
design suggested maybe to call the prop as "fullWidth" instead of stretch

Reply:
@YossiSaadi refresh